core/drivers/remoteproc/stm32_remoteproc.c

```diff
@@ -174,14 +174,14 @@ TEE_Result stm32_rproc_map(uint32_t firmware_id, paddr_t pa, size_t size,
 	return TEE_ERROR_ACCESS_DENIED;
 }
 
-TEE_Result stm32_rproc_unmap(uint32_t firmware_id, paddr_t pa, size_t size)
+TEE_Result stm32_rproc_unmap(uint32_t rproc_id, vaddr_t *va, size_t size)
```

Comment:
this passes a va or a pointer to a va?
I think it should be either `void *va` or `vaddr_t va` here.